db/system_distributed_keyspace.cc

```diff
@@ -179,33 +165,31 @@ future<> system_distributed_keyspace::remove_view(sstring ks_name, sstring view_
             false).discard_result();
 }
 
+/* When managing CDC descriptions, we will require CDL = ONE reads/writes
+ * only if there's only one running node. Otherwise, we'll require CL = QUORUM.
+ */
+static db::consistency_level quorum_if_many() {
+    return gms::get_local_gossiper().get_live_token_owners().size() > 1
```

Comment:
You're right.
I gave a little more thought to this issue, and these are the conclusions I arrived at:

The first node should use CL = ONE for writing its generation.
The second node should also use CL = ONE for writing its generation. The ONE node that we have to receive an `ack` from is the first node. The second node is considered pending at this point, so even if we use CL = ONE, an `ack` from the second node is not enough.
All other nodes (third, fourth etc.) should use CL = QUORUM for writing their generations: there should be at least 2 nodes in the (`live_nodes` - `pending_nodes`) set. If that fails, we shouldn't let this node start anyway.

The first node should use CL = ONE for reading its (first) generation.
The first node should also use CL = ONE for reading the second node's generation: at the moment of read, the second node will be pending (because it's still bootstrapping). The ONE node that we have to receive an `ack` from is the first node (ourselves). And as observed above, the data will be there.
But if it happens that at the moment of reading the second node's generation, the second node will already be in the NORMAL state (so not pending), we should use CL = QUORUM. If we used CL = ONE, it could happen that we only ask the second node, which may not contain the data (it will, but in theory it's not required; also, this case will probably never happen).
The second node should do the same thing for reading the first and the second generation (i.e., CL = ONE when it's bootstrapping, CL = QUORUM when it's normal).
All other nodes (third, fourth etc.) should use CL = QUORUM for reading generations.

Summary: We should use CL = QUORUM if and only if the number of nodes that are not pending is > 1. That is, if and only if the number of NORMAL nodes is > 1.

This will be the algorithm:
```
static db::consistency_level quorum_if_many() {
    auto& gossiper = gms::get_local_gossiper();

    int n = 0;
    for (const auto& ep: gossiper.get_endpoint_states()) {
        n += gossiper.is_normal(ep.first);
    }

    return n > 1 ? db::consistency_level::QUORUM : db::consistency_level::ONE;
}
```